makefile.shade

```diff
@@ -16,9 +16,16 @@ k-standard-goals
 #benchmark-tests .compile
   @{
     var sql = string.IsNullOrEmpty(E("BENCHMARK_SQL")) ? string.Empty : "-q " + E("BENCHMARK_SQL");
-    var scenarios = new string[] { "plaintext", "json", "mvcplaintext", "mvcjson" };
+    var scenarios = new string[]
+    {
+      "-n plaintext",
+      "-n plaintext --connectionFilter Benchmarks.PassthroughConnectionFilter",
```

Comment:
> The existing convention used in the benchmarks app was camelCase for the config key

Hmm, I guess this is one of those cases where things will seem weird one way or another. I was just remarking that seeing a camelCase commandline argument is strange. Best leave it then if this is going to make things more confusing.